substrate/client/network/src/service.rs

```diff
@@ -1455,17 +1455,17 @@ where
 	fn handle_worker_message(&mut self, msg: ServiceToWorkerMsg) {
 		match msg {
 			ServiceToWorkerMsg::GetValue(key) =>
-				self.network_service.behaviour_mut().get_value(key),
+				self.network_service.behaviour_mut().get_value(key.to_vec().into()),
```

Comment:
Why not just `key.into()` here and below?

Reply:
I would say that it's because `service.rs` in general, and `network_service` in particular, are still dependant on `libp2p`.